simple-nft-svelte/contract/simple-nft.py

```diff
@@ -228,9 +230,9 @@ def make_metadata(symbol, name, decimals):
         """Helper function to build metadata JSON bytes values."""
         return sp.map(
             l={
-                "decimals": sp.utils.bytes_of_string("%d" % decimals),
-                "name": sp.utils.bytes_of_string(name),
-                "symbol": sp.utils.bytes_of_string(symbol),
+                "decimals": utils.bytes_of_string("%d" % decimals),
```

Comment:
so now we are in normal python code - not SmartPy code - and you may hear us talking about 'being in a test_scenario' or 'scenario code' or similar, we mean normal python code.  Anyway, the `bytes_of_string` function and `metadata_or_url` function are in `scenario_utils` now - I think this file is a bit old? - so we need `sp.scenario_utils.bytes_of_string`.

But note that is just pure python, not SmartPy.

Reply:
>  think this file is a bit old?
I guess I should rewrite it or use an updated file that uses FA2lib.